packages/migrations/src/clickhouse-actions/011-audit-logs.ts

```diff
@@ -0,0 +1,20 @@
+import type { Action } from '../clickhouse';
+
+export const action: Action = async exec => {
+  await exec(`
+    CREATE TABLE IF NOT EXISTS "audit_log"
```

Comment:
Let's rename it to `audit_logs` to follow the naming strategy of the other tables.